src/Core/Project.fs

```diff
@@ -456,32 +456,38 @@ module Project =
         promise {
             let! dotnet = LanguageService.dotnet ()
             match dotnet with
-            | Some dotnet -> return Process.spawnWithNotification dotnet "" cmd outputChannel
+            | Some dotnet -> return Process.spawnWithNotification dotnet cmd outputChannel
             | None -> return! Promise.reject "dotnet binary not found"
         }
 
     let exec exe outputChannel cmd =
         promise {
-            return Process.spawnWithNotification exe "mono" cmd outputChannel
+            return Process.spawnWithNotification exe cmd outputChannel
         }
 
     let private execWithDotnetWithShell cmd =
         promise {
             let! dotnet = LanguageService.dotnet ()
             match dotnet with
             | Some dotnet ->
-                return Process.spawnWithShell dotnet "" cmd
+                return Process.spawnWithShell dotnet cmd
             | None -> return! Promise.reject "dotnet binary not found"
         }
 
     let private execWithShell exe cmd =
         promise {
-            return Process.spawnWithShell exe "mono" cmd
```

Comment:
Mono 😱
Do we use this function anywhere?

Reply:
dont think so, but i'll check and burn it with fire if we can.

Reply:
oh I already had removed all of this :p